### DISCUSSION TITLE
Drafts/Revisions and Tokenized Preview Links

### DISCUSSION THREAD

**Discussion by msimpson** [solspace/craft-calendar#65] [Feature Requests]
The "View" button on standard Craft entries is an easy way for our client to pass previews of changes among stakeholders.

Are there any plans to implement such a system within the Craft version of the Calendar plugin?

**Comment by kjmartens** [comment]
Correct me if I'm wrong, but I wonder if the *Share* button that Calendar includes is not the same as Craft Entries' *View* button (and they seemingly just renamed it recently)? :)

**Reply by msimpson** [reply]
That would be the button to create a preview link if drafts/revisions were supported.
